packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -469,6 +475,17 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          <div className='form-group'>
+            <label>{_('remoteEncryptionKey')}</label>
+            <input
+              className='form-control'
+              name='encryptionKey'
```

Comment:
We need to update `xo-server/sensitive-values.mjs` to make it obfuscate this property.

(To ensure it is not transmitted to the API clients.)